doc/starting-guide.md

```diff
@@ -30,23 +30,23 @@ You need to have an emulator like [AVD](https://developer.android.com/studio/run
 
 #### Running on a simulator
 
-We highly recommend using the `iPhone 11 Pro` simulator as its screen dimensions match with our design.
+We highly recommend using the `iPhone 13` simulator as its screen dimensions match with our design.
```

Comment:
The old designs are still based on the iPhone 11 Pro screen size. Do you think we should also mention this in the documentation?

Reply:
I agree! Let's highlight this and mention the different sizes we can use for different iPhone screens

Reply:
Sure I can update the docs on specifying various sizes that work.
I refrained from doing so in the past because it can create confusion.

Confusion because whatever was designed for iPhone 11 Pro worked if implemented according to figma looked perfect on iPhone 13.

Hence I initially wanted to keep only iPhone 13 in docs.